pandas/core/ops/__init__.py

```diff
@@ -293,6 +294,85 @@ def fill_binop(left, right, fill_value):
 # Dispatch logic
 
 
+def operate_blockwise(left, right, array_op):
```

Comment:
ok much nicer / readable that before. I would consider moving this to a new module to de-nest these functions for blockwise operations (e.g. pandas/core/ops/blockwise.py), but can be later.

Reply:
i think if you move this to ops/blockwise.py would be a +1

Reply:
would really like to put this in a new file